fence/resources/audit_service_client.py

```diff
@@ -0,0 +1,120 @@
+import flask
+import requests
+import time
+
+from fence.config import config
+from fence.errors import InternalError
+
+
+def get_request_url():
+    request_url = flask.request.url
+    base_url = config.get("BASE_URL", "")
+    if request_url.startswith(base_url):
+        request_url = request_url[len(base_url) :]
+    return request_url
+
+
+def is_audit_enabled(category=None):
+    enable_audit_logs = config.get("ENABLE_AUDIT_LOGS") or {}
```

Comment:
Could we do...
```suggestion
    enable_audit_logs = config.get("ENABLE_AUDIT_LOGS", {})
```

Doesn't really make a difference though. Just curious!

Reply:
the `gen3config` lib should ensure all the fields that are in the default config file will be in `config`. So i could actually do `config["ENABLE_AUDIT_LOGS"]` instead of `config.get("ENABLE_AUDIT_LOGS")`

But if we configure `ENABLE_AUDIT_LOGS:` then this null value will override the default value, so we need `or {}` to be safe

Reply:
👍